lib/pages/athlete/AthletePage.dart

```diff
@@ -987,29 +987,34 @@ class _AthletePageState extends State<AthletePage> {
                       ),
                     ],
                   ),
-                  SizedBox(width: 30),
-                  Container(
-                    width: 100,
+                  //SizedBox(width: 0),
+                  Expanded(
+                  child:Container(
+                    width: 0,
                     child: Container(
                       height: 20,
                       decoration: boxDecoration(
                           Colors.amber[500]!.withOpacity(0.20),
                           500,
                           1,
                           Colors.transparent),
+                      
                       child: TextButton(
                         onPressed: () {
                           webWallet.addTokenToWallet(
                               _getCurrentTokenAddress(), _getTokenImage());
                         },
                         child: Text(
                           "+ Add to Wallet",
+                          
                           style:
                               textStyle(Colors.amber[500]!, 10, false, false),
+                              overflow: TextOverflow.ellipsis
```

Comment:
No we cannot have this `overflow: TextOverflow.ellipsis` We want the text to show so you may have to scale the text size depending on the container size so that it does show up